src/migrations/08-rss.ts

```diff
@@ -0,0 +1,18 @@
+import Knex from "knex";
+
+async function up(knex: Knex.Knex): Promise<void> {
+	await knex.schema.createTable("rss", (table) => {
+		table
+			.integer("indexer_id")
+			.references("id")
+			.inTable("indexer")
+			.primary();
+		table.string("last_seen");
```

Comment:
would there a way to use both guid _and_ datetime? I saw some chatter in #development about releases getting deleted which seems like a valid concern.

Reply:
if it ends up that datetime _is_ just better, sorry for the thrash 😬

Reply:
> would there a way to use both guid _and_ datetime? I saw some chatter in #development about releases getting deleted which seems like a valid concern.

Yes, my concern was that the release we are triggering completion off of could be removed fornsomething earoy in its life, for instance beong banned group, mislabeled, or invalid in some other way.

The result would be full paging back.

I think we could set a time max of like an hour before the previously seen release.

Reply:
It does seem kind of pathological that we could accidentally page back all 10 pages looking for a guid that doesn't exist.

Reply:
Exactly

Reply:
We could check for guid always then fallback to time if missing. Storing the last seen time is still susceptible to timezones changes, though unlikely to happen simultaneously.

Since we would only need to cover the deleted torrent case, we could take the time since last run and subtract that from the current latest candidate as the limit. If the tracker uploads frequently then this time will be perfect. If they update rarely then the first page likely covers the time range.

Reply:
I like that idea.

Reply:
sounds fine

Reply:
Can we call this `last_seen_guid`?